libhb/decavcodec.c

```diff
@@ -1,2326 +1,2335 @@
-/* decavcodec.c
```

Comment:
What's the deal with this file?  Looks like something is making it think every line is different?  EOLM?

Reply:
interesting but applying each patch separately - works just fine

Reply:
It'll work fine. You just can't tell what changed easily. It just hurts the file history if your looking to see what changed.